docs/development/00-developing.adoc

```diff
@@ -144,22 +144,24 @@ $ make test.unit
 
 As of this writing, `dklb`'s end-to-end test suite has the following additional requirements:
 
+* The target DC/OS cluster must be running on AWS.
 * The end-to-end test suite must run from _outside_ the target DC/OS cluster.
+* To test cloud load-balancer provisioning, the ID of a public subnet must be specified using `AWS_PUBLIC_SUBNET_ID`.
```

Comment:
What about the AWS IAM policy needed for managing ELB?

Reply:
I am still not sure what the exact policy is, but I am expecting that to be part of the EdgeLB documentation to which we'll be able to link.